elastio-fargate-s3-backup/README.md

```diff
@@ -0,0 +1,39 @@
+# AWS S3 backup by Elastio and AWS Fargate
+
+## Create AIM Role for Elastio Backup
+
+1. Go to [AWS IAM Roles](https://console.aws.amazon.com/iamv2/home#/roles)
+2. Press Create Role button
+3. Select Elastic Container Service Task under Elastic Container Service for AWS service trusted entity and press Next
+![image](https://github.com/elastio/contrib/assets/81738703/0a7050a0-895b-4227-a609-40bb9c6acb24)
+4. Select ElastioBackupAdmin and AmazonS3ReadOnlyAccess permissions and press Next
+5. Type ElastioS3BackupRole as a role name and press Create role
+
+## Create ECS Task Definition
+
+1. Go to [AWS Elastic Container Service Task definitions](https://console.aws.amazon.com/ecs/v2/task-definitions)
+2. Press Create new task definition
+3. Type ElastioS3Backup as a task definition family name
+4. Select ElastioS3BackupRole as Task role
+5. Type Elastio-CLI as container name
+6. Paste `public.ecr.aws/elastio-dev/elastio-cli:latest` in container image URI
+7. Expand Docker configuration and paste `sh,-c` in Entry point and `elastio s3 backup --bucket <S3 Bucket Name>` in Command. Where `<S3 Bucket Name>` is a name of your bucket for backup
```

Comment:
Is that `,` after `sh` a typo, or it really is a comma-delimited list of arguments?

Reply:
It is comma-delimited list of arguments.